binaryaudit/RPM.py

```diff
@@ -0,0 +1,27 @@
+# Group packages based on source RPM
+
+import json
+import subprocess
+
+def groupPackages(dirPath):        
```

Comment:
This function needs a test please.

Thanks.

Reply:
Thinking more on this, another suggestion yet - please lets move this function into the `abicheck` module and make its signature as follows:

`def generate_package_json(source_dir, out_filename)`

Later there can be also argument to differentiate by package type.

Thanks.